src/main/java/frc/robot/misc/Constants.java

```diff
@@ -33,16 +33,16 @@ public static final class Swerve {
                 new Translation2d(-wheelBase / 2.0, -trackWidth / 2.0));
 
         /* Swerve Voltage Compensation */
-        public static final double voltageComp = 12.0;
+        public static final double motorVoltage = 12.0;
         public static final double thriftyMaxVoltage = 4.8;
         
         /* Swerve Current Limiting */
         public static final int angleContinuousCurrentLimit = 20;
         public static final int driveContinuousCurrentLimit = 40;
         
         /* Angle Motor PID Values */
-        public static final double angleKP = 2;//12;//1.6; //1.6 on our robot
-        public static final double angleKI = 1;//1.8 on our robot
+        public static final double angleKP = 5; //1.6 on our robot
```

Comment:
What do the comments at the end of the line mean? Just clean them up if they aren't meaningful. If they're important, explain better than "on our robot."